OCS/deploy-ocs.sh

```diff
@@ -4,15 +4,15 @@ LOCALNAMESPACE="local-storage"
 
 # export is required for envsubst
 export rook_override="${rook_override:-false}"
-export public_network="${public_network:-1.1.1.1/24}"
-export cluster_network="${cluster_network:-1.1.1.1/24}"
+export public_network="$(ip r | awk '/dev\ baremetal\ proto\ kernel\ scope\ link\ src/ {print $1}')"
+export cluster_network="$(ip r | awk '/dev\ provisioning\ proto\ kernel\ scope\ link\ src/ {print $1}')"
```

Comment:
According to https://docs.google.com/document/d/1RU0KcS1EdY1JKrjLbHhk5qRhMmHsbbCkeOy2vDF1e4g/edit?usp=sharing

The storage cluster will  be s separate network.   

Also, since this looke to be runing on the deply node, doesn't this assume the deply node is the same as the masters?
Thoght that was an issue in the virtual world.

Reply:
It shouldn't be a problem on the virtual environment since the subnets are the same, the only difference compared to the bare metal envs is in the nic names.

Regarding the storage network afaik we are still not creating it on the deployment yet so my assumption was that the provisioning network is used as the cluster_network atm.